### PR TITLE
DEV-281 - Fix server semver parsing for CI

### PR DESCRIPTION
**Pull request by josephcummings**
Fixed: parsing of server semver for CI, where the server version may have tagging

### CODE REVIEW

File: db-client-java/build.gradle [hunk 1]
```diff
@@ -136,7 +136,10 @@ test {
 protobuf {
     protoc {
         // The artifact spec for the Protobuf Compiler
-        artifact = 'com.google.protobuf:protoc:3.13.0'
+        artifact = osdetector.arch == "aarch_64"
```
Comment (josephcummings) [inline db-client-java/build.gradle]: Pull into separate PR, seems we can upgrade to access arm64 binaries

Comment (josephcummings) [reply db-client-java/build.gradle]: Moved to #265

**Review by w1am — APPROVED** [review]

**Review by w1am — APPROVED** [review]
